development/testing/runtest.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/env bash
+#!/bin/bash
```

Comment:
`#!/bin/env bash` <- it's better to use this as it takes the bash shell from the env rather than from /bin/bash as not everyone could have their shell there - or something like that

Reply:
`#!/usr/bin/env bash` <- This is better